packages/lit-dev-content/site/tutorials/content/reactivity/05.md

```diff
@@ -0,0 +1,53 @@
+Sometimes you need to do something after the component has updated. For example, you may need to measure the component after the update, in order to perform an animation. There's a callback for that, and unsurprisingly it's called `updated()`.
+
+The component on the right displays a message when you click the button. It's not very exciting, though. It could use some kind of transition.
+
+Add the following `updated()` method to the component:
+
+{% switchable-sample %}
+
+```ts
+  protected updated(changedProperties: PropertyValues<this>): void {
+    if (changedProperties.has('_showMessage')) {
+      const final = this._message.getBoundingClientRect().width;
+      const starting = 0 - final;
+      var player = this._message.animate([
+        { left: `${starting}px` },
+        { left: `0` }
+      ], {
+        duration: 500,
+        easing: 'ease-out',
+      });
+    }
+  }
+```
+
+```js
+  updated(changedProperties) {
+    if (changedProperties.has('_showMessage')) {
+      const final = this._message.getBoundingClientRect().width;
+      const starting = 0 - final;
+      var player = this._message.animate([
+        { left: `${starting}px` },
+        { left: `0` }
```

Comment:
Consider using transform: translateX since it causes less jank in the demo

```suggestion
        { transform: `translateX(${starting}px)` },
        { transform: `translateX(0)` }
```